### ISSUE TITLE
Clarification needed about external RAM size nv_small 

### ISSUE THREAD

**Issue by Y159753** [nvdla/hw#169]
I am trying to figure out what is the minimal size for the the external RAM in nv_small : 
- in "Scalability parameters and ConfigROM" for nv_small it only needs 56KBytes
http://nvdla.org/hw/v2/scalability.html#address-space-layout
- but how much space is needed usually for storing in RAM during execution of UMD, KMD and runtime application ?
- as well do you have an example of size of a neural net loadable + its test images.

**Comment by Y159753** [comment]
I just discovered that the cfgROM is inside the NVDLA and not outside.

**Comment by nvhook** [comment]
@Y159753 Issue resolved?

**Comment by Y159753** [comment]
Not really, but I will not synthesize the external RAM inside a chip anymore. So problem solved
